src/thebelab.js

```diff
@@ -248,13 +248,17 @@ function renderCell(element, options) {
   Widget.attach(outputArea, theDiv);
 
   const mode = $element.data("language") || "python";
+  const isReadOnly = $element.data("readonly") || false;
```

Comment:
Should `data-readonly=""` and `data-readonly` (which are viewed equivalently in JS) be considered readOnly or not? This determines if this should be `||` falsy or `??` nullish coalescing.

Reply:
The convention I can find suggests that for booleans like this, it should be `<pre data-readonly>...` to be considered true, so the only false values ought to be `undefined` and (maybe) explicit `false`:

```html
<pre> # false
<pre data-readonly='false'> # false
<pre data-readonly> # true
<pre data-readonly='true'> # true
```

I *would* say that we should follow the boolean-only convention and use

```python
const isReadOnly = $element.data("readonly") !== undefined;
```

but the fact that we already have `data-executable="true"` (which I now realize was maybe a mistake), suggests we should support true/false to be consistent with ourselves.

In the future, we should probably add support for this no-right-hand-side convention in data-executable as well, to be consistent.